infrastructure/main.tf

```diff
@@ -102,4 +107,19 @@ resource "azurerm_key_vault_secret" "staging_db_database" {
   value        = module.reform-notifications-staging-db.postgresql_database
 }
 
-# endregion
\ No newline at end of file
+# endregion
+
+# region Copy secrets from BulkScan
+
+data "azurerm_key_vault_secret" "bulk_scan_processor_s2s_secret" {
+  key_vault_id = "${data.azurerm_key_vault.bulk_scan_key_vault.id}"
+  name         = "s2s-secret"
+}
+
+resource "azurerm_key_vault_secret" "bulk_scan_processor_s2s_secret_copy" {
```

Comment:
can be bulk_scan_processor_s2s_secret_copy  -> test-bsp-s2s-secret as we do in other repos but not too strong about it